### PR TITLE
NPT-609 Add JIRA creation from actions when issue is created

### PR DESCRIPTION
**Pull request by ramramu3433**
* Adding Workflow trigger when the issue is opened
* workflow uses secrets to connect to JIRA and comment on the JIRA ID in the issue after JIRA creation.
* close_jira.py is added will work on auto transition JIRA to Done 
* JIRA will only be created for bugs(Bug) and feature requests(Task).

### CODE REVIEW

File: close_jira.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+from jira import ( JIRA,JIRAError )
```
Comment (mkczuk) [inline close_jira.py:1]: is this file used? or do we plan to use in the future?

File: close_jira.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import os, sys, requests, re, json
+
+issue_types = [{"Bug":"bug"},{"Task":"feature request"}]
+def parse_event_context():
+    """
+    Parse event context and run issue creation checks
+    """
+    try:
+        create_issue_for = False
```
Comment (mkczuk) [inline close_jira.py:10]: delete_issue_for seems to make more sense here

File: close_jira.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
+from jira import ( JIRA,JIRAError )
+import os, sys, requests, re, json
+
+issue_types = [{"Bug":"bug"},{"Task":"feature request"}]
+def parse_event_context():
```
Comment (mkczuk) [inline close_jira.py:5]: nit: consider moving this parser to seprate file to avoid duplication

File: close_jira.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        'summary': issue_title ,
+        'description': issue_body,
+        'issuetype': {'name': issue_type },
+    }
+    try:
+        URL=issue_url+"/comments"
+        github_token = os.environ.get("GITHUBTOKEN")
+        resp = requests.get(URL,headers={"Authorization":"Bearer {}".format(github_token),"Accept":"application/vnd.github+json"})
+        response = json.loads(resp)
+        for comment in response:
```
Comment (mkczuk) [inline close_jira.py:64]: I'm not sure if I understand this code correctly, but it looks for jira links mentioned in comments and closes first one which was found? so we consider first one which will be found will be a link to jira which we autocreated, right? is this bulletproof solution? if someone would just mention some other ticket in description or comment wouldn't it close it?

File: close_jira.py (hunk 1 above)
Comment (ramramu3433) [reply close_jira.py]: removing this file as closing JIRA requires more thought process

**Review by mkczuk — APPROVED** [review]

**Pull request merged**